### PR TITLE
Fix bug with send-science permission

### PR DESCRIPTION
**Pull request by clifffrey**
I think that the only bug is with the "shortcut" send-science button. However, I added the check in the on_gui_click handlers just to be extra-super-sure.

### Tested Changes:
- [x] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

File: comfy_panel/special_games/captain.lua [hunk 1]
```diff
@@ -3306,14 +3306,15 @@ function Public.clear_gui_special()
 end
 
 function Public.captain_is_player_prohibited_to_throw(player)
```
Comment (RedRafe) [inline comfy_panel/special_games/captain.lua:3308]: I would prefer this function to return a false at the end of the if cascade, just to be clear this is supposed to be a bool function and not a nil|true function

**Review by RedRafe — APPROVED** [review]
Just a minor comment on the cpt function

File: comfy_panel/special_games/captain.lua (hunk 1 above)
Comment (clifffrey) [reply comfy_panel/special_games/captain.lua:3308]: I am confused. It already has `return false` as the last line of the function on line 3321. You want me to add that additionally after line 3319?

Comment (RedRafe) [reply comfy_panel/special_games/captain.lua:3308]: Nah sorry I had the editor trimming just before that line. That was exactly what I was expecting, nvm :)

**Pull request merged**